src/python/pants/help/help_info_extracter.py

```diff
@@ -879,6 +890,25 @@ def load() -> BackendHelpInfo:
             }
         )
 
+    @classmethod
+    def get_build_file_info(
+        cls, build_symbols: BuildFileSymbolsInfo
```

Comment:
```suggestion
    @staticmethod
    def get_build_file_info(
        build_symbols: BuildFileSymbolsInfo
```
Does this need to be a class method?

Reply:
nope, guess not :P